docs/common/project/how-to-release.md

```diff
@@ -60,6 +60,11 @@ To use `gpg` instead of `gpg2`, also set `signing.gnupg.executable=gpg`
 
 For more information, see the Gradle [signing documentation](https://docs.gradle.org/current/userguide/signing_plugin.html#sec:signatory_credentials).
 
+### Apache repository
+
+The release should be executed in the official `https://github.com/apache/iceberg.git` instead of a forked one.
+Clone the Apache Iceberg repository or set it as a different origin for release if it is not already set up.
```

Comment:
"origin" should be "remote" right? Origin is typically a specific remote name (your own private fork).

Reply:
correct, let me just say setup a new remote called `apache` here